lib/test/karma.js

```diff
@@ -119,7 +203,38 @@ function karmaRun(bannerWidth) {
   });
 };
 
+var yargsOptionDefiniton = {
+  key: 'karmareporter',
+  value: {
+    describe: 'Specify a custom Karma reporter to use. '+
+      'Either a locally npm installed module, or an asolute path to one.',
+    alias: ['k'],
+    default: defaultReporterName,
+    string:true,
+  }
+};
+var checkKarmaReporter = yargs.createCheck()
+  .withGate(function (argv) {
+    return !argv.help;
+  })
+  .withTest({
+    'karmareporter': function(value) {
+      if (typeof value === 'undefined') {
```

Comment:
value is always defined so there is no value in anything other than
`if (value === undefined) {`